src/Handler/Database.hs

```diff
@@ -33,6 +34,19 @@ getAllPackageNames = do
   contents <- liftIO $ getDirectoryContents (dir ++ "/verified/")
   return . sort . rights $ map mkPackageName contents
 
+getLatestPackages :: Handler [(PackageName, Version)]
+getLatestPackages = do
+    pkgNames <- getAllPackageNames
+    pkgNamesAndVersions <- traverse withTimestamp pkgNames
```

Comment:
I might be missing something but could this identifier's name be a little misleading?